renovate.json

```diff
@@ -685,6 +685,24 @@
         "backport:prev-minor"
       ],
       "enabled": true
+    },
+    {
+      "groupName": "machine learning modules",
+      "matchPackageNames": [
+        "apidoc-markdown"
+      ],
+      "reviewers": [
+        "team:ml-ui"
+      ],
+      "matchBaseBranches": [
+        "main"
+      ],
+      "labels": [
+        "Team:ML",
+        "release_note:skip",
+        "backport:all-open"
```

Comment:
Just for my own understanding, do we typically default to `backport:all-open`, and deescalate as needed?

Reply:
> do we typically default to backport:all-open, and deescalate as needed?

Correct. The effect of not back-porting to the previous major accumulates over time, which might become problematic once we absolutely have to backport something that depends on previous upgrades that weren't backported previously. Even though there is always a risk of breaking something in the old branch with technically unnecessary backporting, it would still be easier to isolate and debug the issue if we backport it incrementally.

Does that make sense?